rules/wiz_rules/wiz_alert_passthrough.py

```diff
@@ -13,11 +13,15 @@ def title(event):
 
 
 def severity(event):
+    if event.get("severity") == "INFORMATIONAL":
```

Comment:
Doesn't the `if` check in the `rule` function mean this scenario will never occur?

Reply:
I commented out the unused code, but I think it's worth leaving as a reminder that the Wiz INFORMATIONAL severity does not map to our INFO severity.